src/components/avatar/Avatar.css

```diff
@@ -0,0 +1,64 @@
+:root{
+    --dark-blue:rgb(29, 29, 46);
+    --gray:#999999;
+    --dark-green:rgb(49, 94, 75);
```

Comment:
if possible stick to consistent use of color values, use hex instead of rgb and for translucent colors you can use rgba.